REFERENCE.md

```diff
@@ -4177,12 +4178,19 @@ Default value: `undef`
 
 Data type: `Optional[Variant[String, Array]]`
```

Comment:
Shouldn't this be switched to `Optional[String]`  now? I don't see any other options except the log level here https://nginx.org/en/docs/ngx_core_module.html#error_log

Reply:
As far as I understood, this is to be able to specify multiple lines of error_log. Look at the templates.

Reply:
Right I did overlook that. The template is kind of ugly, I think it would benefit from a refactoring some time. Maybe it would make sense to change `error_log` to a more specific type to aid in that. Something like this:

```
Variant[Undef, 'absent', Array[Hash[Stdlib::Absolutepath,Optional[Nginx::ErrorLogSeverity]]] = [
  {   '/foo/bar' => 'undef' },
  {   '/a/b/'       => 'warn' },
}
```
Just some random idea ;)